src/actions/articleActions.js

```diff
@@ -0,0 +1,42 @@
+import axios from 'axios';
+import config from '../config';
+
+/**
+ * @class ArticleActions
+ */
+export default class Article {
+  /**
+  * Request to the API to get list of articles
+  *
+  * @static
+  * @returns {Object} dispatch object
+
+  * @memberof ArticleActions
+  */
+  static fetchArticle() {
+    return (dispatch) => {
+      dispatch({ type: 'GET_ARTICLES_BEGINS' });
+      return axios.get(`${config.apiUrl}/articles`)
+        .then((response) => {
+          dispatch({
+            type: 'GET_ARTICLES_SUCCESSFUL',
+            articles: response.data.articles,
+            pageCount: response.data.paginationMeta.pageCount
```

Comment:
You should pass all of the `paginationMeta` to the reducer as they will be useful when carrying out pagination